packages/expo-cli/src/commands/optimize.ts

```diff
@@ -48,19 +48,19 @@ export default function(program: Command) {
   program
     .command('optimize [project-dir]')
     .alias('o')
-    .description('Compress the assets in your Expo project')
-    .option('-s, --save', 'Save the original assets with a .orig extension')
+    .description('Compress the assets in your Expo project.')
+    .option('-s, --save', 'Saves the original assets with a .orig file extension.')
     .option(
-      '--quality [number]',
-      'Specify the quality the compressed image is reduced to. Default is 80'
+      '--quality [num]',
```

Comment:
```suggestion
      '--quality [number]',
```
(I try to avoid abbreviations in code and documention, especially non-standard abbreviations.)